versioned_docs/version-3.0.x/administration-geo.md

```diff
@@ -227,6 +227,12 @@ If you want to use replicated subscriptions in Pulsar:
               .replicateSubscriptionState(true)
               .subscribe();
   ```
+:::note
+
+* Replication type is not replicated between clusters but determined by the first consumer that connects to the subscription.
```

Comment:
```suggestion
* Replication type is not replicated between clusters. However, it's determined by the first consumer that connects to the subscription.
```
Can we break this long sentence to improve readability?

Reply:
yes